test/ecto/migration_test.exs

```diff
@@ -1004,6 +1021,23 @@ defmodule Ecto.MigrationTest do
       assert {:create, %Index{}} = last_command()
     end
 
+    test "renames an index" do
+      rename index(:people, [:name]), to: "person_names_idx"
+      flush()
+      {_, index, old_name} = last_command()
+      assert old_name == :people_name_index
```

Comment:
It is unclear to me why the third element of the `last_command/0` return tuple is a string in the forward direction and an atom in the reverse direction.

Reply:
Oh, duh. The one direction is automatically generated from the table and column name, the other I specified as a string.